backend/fms_core/template_importer/row_handlers/sample_submission/sample.py

```diff
@@ -16,6 +16,9 @@
 from fms_core.services.index import get_index
 
 class SampleRowHandler(GenericRowHandler):
+    HUMAN_TAXON_ID = 9606
+    MOUSE_TAXON_ID = 10090
```

Comment:
Erm... You sure it's good in the long run to hardcode those?

Reply:
Nah i am pretty sure it's not great i just have a hard time finding a good way around ... it all comes down to harcoding human and mouse in some form ... I could add another attribute to the taxon to indicate they are handle differently but it would be just a more ugly (DB hardcoding ?) solution in my eyes unless we can make in generic somehow ...